### DISCUSSION TITLE
How to style toast?

### DISCUSSION THREAD

**Discussion by GavanWilhite** [chakra-ui/chakra-ui#5895] [Q&A]
I cannot figure out how to correctly style the toast notification. It seems like a random subset of style gets passed on from the theme alert / toast settings, but with weird changes like extra transparency and nothing seems to change the text color.

Any help?

**Comment by sudazzle** [comment]
I hacked with old school css.
<style>
      {`.chakra-toast .chakra-alert {
      padding: 20px;
    }`}
</style>

You can also tweak Alert component theme as well I think as it uses Alert component under the hood.

But it would be nice to pass down to useToast hook like the containerStyles prop.

**Comment by ghost** [comment]
I have the same issue. If you have a look in here: https://github.com/chakra-ui/chakra-ui/blob/main/packages/components/toast/src/toast.tsx you can see the call being made to the Alert component behind the scenes when using the toast. In my case I would like to use a different border radius than what is defined as md in my custom theme, but it seems that is not possible. Without having inspected the helper functions in detail, I assume any styling passed as props take precedence over theme based styling, so other than hacking this with some custom css you cannot change the styling of the ones that are hardcoded in the Alert component in the linked file.

**Reply by jinsley8** [reply]
Add the alert component override to your theme. 

```js
// alert.ts

import { alertAnatomy } from '@chakra-ui/anatomy';
import { createMultiStyleConfigHelpers } from '@chakra-ui/react';

const { definePartsStyle, defineMultiStyleConfig } = createMultiStyleConfigHelpers(alertAnatomy.keys);

const baseStyle = definePartsStyle({
    container: {
        //.... container styles
    },
    title: {
        //.... title styles
    },
    description: {
        //.... description styles
    },
    icon: {
        //.... icon styles
    },
});

export default defineMultiStyleConfig({ baseStyle });
```

**Reply by TheDuckGoesQuark** [reply]
This works bar styling the close button when `closeable` is `true`, but solves most other problems
